app/src/main/java/com/codingblocks/cbonlineapp/activities/NotificationsActivity.kt

```diff
@@ -70,6 +70,24 @@ class NotificationsActivity : AppCompatActivity() {
             onClick = eventClickListener
         }
 
+        val itemTouch = object : ItemTouchHelper.SimpleCallback(0, ItemTouchHelper.LEFT or ItemTouchHelper.RIGHT){
+            override fun onMove(recyclerView: RecyclerView, viewHolder: RecyclerView.ViewHolder, target: RecyclerView.ViewHolder): Boolean {
+                return false
+            }
+
+            override fun onSwiped(viewHolder: RecyclerView.ViewHolder, direction: Int) {
+                val position = viewHolder.adapterPosition
+                // get all notifications
+                val notifications = notificationDao.allNotification.value
+                // get the id of element which needs to be deleted
+                val deleteUID = notifications?.get(position)?.id
```

Comment:
Position and id will not be same here and I'm not sure whether this will work or not for all the cases

Reply:
position is index of notification object in the list and id is the UID of that notification in the list. It will work as each notification is going to have unique ID.
Here is the demo video - https://drive.google.com/open?id=1wj2TSoXDPwG03BJUjmpd8lf0W2zGo0iK

@aggarwalpulkit596